Mathlib/Algebra/Ring/UnitEquivs.lean

```diff
@@ -0,0 +1,69 @@
+/-
+Copyright (c) 2021 Johan Commelin. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Eric Rodriguez, Johan Commelin
+-/
+import Mathlib.GroupTheory.GroupAction.ConjAct
+import Mathlib.RingTheory.Subring.Basic
+
+/-!
+# Equivalences relating to units
+
+This file makes a couple equivalences that deal with unit groups under various other operations,
+such as centralizers and stabilizers. These are mainly useful to prove Wedderburn's little theorem.
+
+-/
+
+variable {D : Type*}
+
+/-- The stabilizer of `Dˣ` acting on itself by conjugation at `x : Dˣ` is exactly the
+units of the centralizer of `x : D`. -/
+def unitsCentralizerEquiv [Monoid D] (x : Dˣ) :
+    MulAction.stabilizer (ConjAct Dˣ) x ≃* (Submonoid.centralizer ({↑x} : Set D))ˣ where
```

Comment:
I assume there's not a more general results hiding here for `MulDistribMulAction`s?

Reply:
I'm not even sure what it'd look like